### PR TITLE
Fixed  Stratis.Bitcoin.Features.Consensus.Tests.Rules.CommonRules

### PR DESCRIPTION
**Pull request by bokobza**

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/RulesRegistrationOrderTest.cs [hunk 1]
```diff
@@ -54,9 +57,10 @@ public void StratisMainPowConsensusRulesRegistrationTest()
         [Fact]
         public void StratisMainPosConsensusRulesRegistrationTest()
         {
-            new FullNodeBuilderConsensusExtension.PosConsensusRulesRegistration().RegisterRules(this.network.Consensus);
+            Network network = new StratisTest();
```
Comment (Aprogiena) [inline src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/RulesRegistrationOrderTest.cs]: This is weird change. The test name specifically includes Stratis main and you're now using Stratis test.?

File: src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/RulesRegistrationOrderTest.cs [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 namespace Stratis.Bitcoin.Features.Consensus.Tests.Rules.CommonRules
 {
-    public class ConsensusRulesRegistrationTest : TestConsensusRulesUnitTestBase
+    public class ConsensusRulesRegistrationTest
     {
         [Fact]
         public void StratisMainPowConsensusRulesRegistrationTest()
         {
-            new FullNodeBuilderConsensusExtension.PowConsensusRulesRegistration().RegisterRules(this.network.Consensus);
+            Network network = new BitcoinTest();
```
Comment (Aprogiena) [inline src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/RulesRegistrationOrderTest.cs]: Same here – the network does not match the test name

File: src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/RulesRegistrationOrderTest.cs (hunk 1 above)
Comment (bokobza) [reply src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/RulesRegistrationOrderTest.cs]: ah yes true, didn't see that.
There is no difference between the two so I'll remove Main from the test names.

**Comment by bokobza** [comment]
@zeptin at the moment, the rules are the same. If we introduce new rules specific to a particular network, we will definitely add a test for it.

**Pull request merged**